### PR TITLE
Bumped angular-patternfly to v4.14.6, moment to v2.19.3, and patternfly from to v3.37.4

### PR DESCRIPTION
**Pull request by dtaylor113**
This PR updates angular-patternfly, moment, and patterfly to latest versions in order to (a) show content through filter panel, and (b) remove `unsafe:javascript.void()` from 'Clear Filters' link.

Fixes https://github.com/openshift/origin-web-catalog/issues/536
Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1533363

Patternfly changes were in wizards, notification drawer, and toast notifications.  Visually compared and tested each, no issues detected.

This also addresses a Node Security issue in previous version of `moment`:
image

### CODE REVIEW

**Comment by spadgett** [comment]
Since we only use moment on the client-side, the server-side denial of service shouldn't impact the console.

/assign @rhamilto 

Robb, can you take a look?

File: app/index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
     <link rel="stylesheet" href="bower_components/kubernetes-container-terminal/dist/container-terminal.css" />
     <link rel="stylesheet" href="bower_components/registry-image-widgets/dist/image-widgets.css" />
     <link rel="stylesheet" href="bower_components/layout.attrs/dist/layout.attrs.css" />
+    <link rel="stylesheet" href="bower_components/bootstrap-slider/dist/css/bootstrap-slider.css" />
```
Comment (rhamilto) [inline app/index.html]: This vendor CSS isn't necessary as it is included in PatternFly, so add `bower_components/bootstrap-slider/dist/css/bootstrap-slider.css` to the Gruntfile ignore.

Comment (dtaylor113) [reply app/index.html]: Fixed, thanks.  I'll keep an eye on 'bower_components/bootstrap-slider/dist/bootstrap-slider.js', it shouldn't be included either but waiting for patternfly core version release which includes bootstrap-slider in bower

**Comment by rhamilto** [comment]
It looks like the [ReDoS fix does apply to moment.js](https://github.com/moment/moment/pull/4326/files#diff-8c9255ccee4354533dcec358fb36d701L662), so we need to upgrade.

**Comment by dtaylor113** [comment]
HI @rhamilto, I believe I have addressed all of your issues.  -thanks

**Pull request merged**